lib/rex/post/meterpreter/client_core.rb

```diff
@@ -431,7 +431,7 @@ def get_ssl_hash_verify
   # Migrates the meterpreter instance to the process specified
   # by pid.  The connection to the server remains established.
   #
-  def migrate(pid, writable_dir = nil)
```

Comment:
Changing the calling syntax of ``migrate()`` may break scripts and post-modules.